controllers/storagecluster/exporter.go

```diff
@@ -709,3 +715,86 @@ func createMetricsExporterRolebindings(ctx context.Context, r *StorageClusterRec
 
 	return nil
 }
+
+func createRookCephClusterRolebindings(ctx context.Context,
+	r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
+	// rookCephMonitorMgrRoleBinding is a cluster rolebinding for monitor mgr
+	rookCephMonitorMgrRoleBinding := &rbacv1.ClusterRoleBinding{
```

Comment:
```suggestion
	rookCephMonitorMgrRoleBinding := rbacv1.ClusterRoleBinding{
```
Since we're just copying these values later, do we need this to be a pointer?

Reply:
Nope, we don't require any pointers.